src/typescript/frontend/src/components/trade/OrderEntry/LimitOrderEntry.tsx

```diff
@@ -60,6 +64,76 @@ export const LimitOrderEntry: React.FC<{
     };
     const orderSide = orderSideMap[side];
 
+    // validation
+    const _rawValueSize = toRawCoinAmount(
+      values.size ?? 0,
+      marketData?.base?.decimals ?? 0,
```

Comment:
Best to check whether you have all the values you need at the top instead of using optional chaining and null coalescing everywhere. Performing validation using 0 if the values here are undefined is kinda pointless

Reply:
good point